source/_posts/2024-04-02-open-source-community-cra-compliance-initiative.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Joining Forces for Open Source Cybersecurity Standards'
+layout: post
+tags: 
+  - news
+author:
+  name: Roman Pronskiy
+  url: https://twitter.com/pronskiy
+published_at: 02 March 2024
+---
+
+The PHP Foundation, alongside leading open source organizations including the [Apache Software Foundation](https://www.apache.org/), [Blender Foundation](https://www.blender.org/about/foundation/), [OpenSSL Software Foundation](https://www.openssl.org/), [Python Software Foundation](https://www.python.org/psf-landing/), [Rust Foundation](https://foundation.rust-lang.org/), and [Eclipse Foundation](https://www.eclipse.org/), is excited to announce a collaborative initiative aimed at establishing common cybersecurity standards in alignment with the European Union’s [Cyber Resilience Act (CRA)](https://digital-strategy.ec.europa.eu/en/library/cyber-resilience-act). 
+
+This initiative, hosted by the Eclipse Foundation, will focus on creating specifications for secure software development rooted in open-source best practices.
+
+> We hope that our specifications could inform the formal standardisation processes of at least one of the European Standards Organisations. Given the tight time horizon for implementation of the CRA, we believe that this immediate start will provide a constructive environment to host the technical discussions necessary for the stewards, contributors, and adopters of open source to meet the requirements set forth in these new regulations.
+
+At The PHP Foundation, we are committed to supporting the open-source community, ensuring that PHP language and its ecosystem are prepared for the upcoming regulatory changes.
```

Comment:
Inconsistency: here it's `open-source`, earlier it's `open source`.

I think `that PHP language` should be `that the PHP language`.